src/models/basicstructural_multivariate.jl

```diff
@@ -45,11 +45,26 @@ mutable struct MultivariateBasicStructural <: StateSpaceModel
     end
 end
 
+function handle_multivariate_names(p::Int)
```

Comment:
```suggestion
function handle_multivariate_names(p::Int, ::Type{MultivariateBasicStructural})
```

I would add something like this just to avoid issues with this function being used in other models since it's exclusive to this one